### DISCUSSION TITLE
Wagmi docs for older versions

### DISCUSSION THREAD

**Discussion by c0rdeiro** [wevm/wagmi#3391] [General]
Shouldn't we have the older version docs available in the website (v 1.x.x), similar to viem?

**Comment by tmm** [comment]
It exists! https://1.x.wagmi.sh

**Reply by c0rdeiro** [reply]
thanks, I just did not found any reference to it in the new website

**Reply by tmm** [reply]
It's on this page https://wagmi.sh/react/guides/migrate-from-v1-to-v2
